Bluepill-runner/Bluepill-runner/BPReportCollector.h

```diff
@@ -20,4 +20,10 @@
              onReportCollected:(void (^)(NSURL *fileUrl))fileHandler
                   outputAtPath:(NSString *)finalReportPath;
 
++ (void)collectReportsFromPath:(NSString *)reportsPath
+                 withOtherData:(NSDictionary *)otherData
```

Comment:
This is very helpful information to add. But "withOtherData" is not easy to understand, maybe "withTestConfig"?